src/app.routes/index.routes.js

```diff
@@ -1,13 +1,16 @@
 const app = require("../../app");
+const PostRoute = require("../posts/posts.routes");
 const UserRoute = require("../users/users.routes");
 
 const basePath = "/api/v1";
 
 module.exports = (app) => {
+    app.use(`${basePath}/users/posts`, PostRoute);
```

Comment:
For simplicity's sake make line 8 to look like this
` app.use(`${basePath}/posts`, PostRoute);`

There are other ways to do this but this is just to keep things simple.
For learning's sake this is how it should have looked using Restful API principles
```
/users
/users/:userId/posts
```

Reply:
Taken note.